library/common/engine.cc

```diff
@@ -284,4 +288,14 @@ void Engine::drainConnections() {
   server_->clusterManager().drainConnections();
 }
 
+void Engine::logInterfaces() {
+  auto name_vec = Network::MobileUtility::enumerateInterfaces();
+  std::string names = std::accumulate(name_vec.begin(), name_vec.end(), std::string{},
+                                      [](std::string acc, std::string next) {
+                                        return acc.empty() ? next : std::move(acc) + "," + next;
+                                      });
+  ENVOY_LOG(debug, "available interfaces: {}", names);
+  ENVOY_LOG_EVENT(debug, "socket_selection_available_interfaces", names);
```

Comment:
just need ENVOY_LOG_EVENT, it "dual writes" to the event tracker and the process logs

Reply:
Yep, I do see that in the log output now, though it would be nice if the raw log outputted the name as well, I think.